### DISCUSSION TITLE
What is the `.ignore` file for?

### DISCUSSION THREAD

**Discussion by hasezoey** [tramhao/termusic#257] [Q&A] [answered]
I have noticed there is a [`.ignore` file](https://github.com/tramhao/termusic/blob/e85f574a48befbf4a53560ec8a2200a350c987dd/lib/.ignore), but i dont know for which program this is for. one thing i know is that it is *not* for git, as git always uses `.gitignore`.

@tramhao could you maybe clarify what that file is for?

**Comment by tramhao** [comment, marked as answer]
That is for helix editor
